src/Cake.Common.Tests/Unit/XML/XmlPeekTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.IO;
+using Cake.Common.Tests.Fixtures;
+using Xunit;
+
+namespace Cake.Common.Tests.Unit.XML
+{
+    public sealed class XmlPeekTests
```

Comment:
Can do. Since I'd only been basing things on XmlPokeTests, I didn't realize it, and my derived code, diverged from the preferred conventions.

I made changes trying to match what you describe in [this commit](https://github.com/patridge/cake/commit/3c174c13e35ec06a0b0010c6e038f47fd6dd8b80).